lockfile.json

```diff
@@ -191,9 +191,6 @@
   "make": {
     "revision": "a4b9187417d6be349ee5fd4b6e77b4172c6827dd"
   },
-  "markdown": {
-    "revision": "ab15701d8f3f68aeb74e30573b7d669a6ef2a7ed"
-  },
```

Comment:
Don't need to remove this; the "Update Lockfile" PR will take care of that (I think), and then you don't have to worry about conflicts or constantly rebasing this file.

Reply:
but the SHA has to be changed in case the new queries are incompatible with the old parser

Reply:
...which they are

but wouldn't this only be a problem for the short window between merging this and the subsequent Update Lockfile PR?

Reply:
> but the SHA has to be changed in case the new queries are incompatible with the old parser

Should I add this back then? Or change it to the new has?

Reply:
Ideally, yes, always adapt the lockfile to parser changes -- saves PR churn, if nothing else.